### DISCUSSION TITLE
help with react-native lesson 6.3

### DISCUSSION THREAD

**Discussion by haikeyidesu** [sst-inc/help#56] [Q&A - React] [answered]
I have managed to get my app working, however, the title does not appear at the topbar of the app. Does anyone know how to fix this? Thanks.

**Comment by ghost** [comment, marked as answer]
Please provide some code (eg. App.tsx)

**Reply by haikeyidesu** [reply]
here

**Reply by haikeyidesu** [reply]
.

**Reply by ghost** [reply]
Actually I think I found the error:
In the HomePage.options thing change `topbar` to `topBar`

**Reply by haikeyidesu** [reply]
thanks it worked
